fedmsg/consumers/__init__.py

```diff
@@ -190,7 +191,10 @@ def _backlog(self, data):
 
             if message['msg_id'] != last['msg_id']:
                 retrieved = retrieved + 1
-                self.incoming.put(dict(body=message, topic=message['topic']))
+                if (self.skip_last_message and retrieved > 1):
+                    self.incoming.put(dict(body=message, topic=message['topic']))
+                else:
```

Comment:
I'm not sure if this will work as intended, it will just skip every message if the `self.skip_last_message` is set to `false`, because the `if` statement  will be true only if `self.skip_last_message` is set to `true` and `retrieved > 1`, everything else will just skip messages. Which I think is not what you want with this PR.

Reply:
Thanks for reviewing. I pledge tunnel vision. I was focused on getting messages dropped. What you say is true. Luckily, _skip_last_message_ is currently set to _True_. :smile: 

Will fix and update this PR and the live environment.